pkg/transport/transport_test.go

```diff
@@ -252,3 +254,18 @@ func TestDialNode_BadServerCertificate(t *testing.T) {
 		assert.Contains(t, err.Error(), "not signed by any CA in the whitelist")
 	})
 }
+
+func TestGetIP(t *testing.T) {
+	conn, err := net.Dial("tcp", "bootstrap.storj.io:8888")
+	require.NoError(t, err)
```

Comment:
This test will be dependent on an external server. I don't know if it is a good idea to use non-local hosts here. If bootstrap.storj.io goes down, it means our tests will fail (locally and on jenkins)

Reply:
Hmm, yeah. Maybe we should remove this